components/Dropdown/index.d.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+export type ItemPropType = string | { value: string | number; label: string };
+
+export interface DropdownProps {
+    autocomplete?: boolean;
+    disabled?: boolean;
+    required?: boolean;
+    error?: string;
+    id?: string;
+    label?: string;
+    placeholder?: string;
+    selectedItem?: ItemPropType;
+    helperText?: string;
+    /** More about stateAndHelpers parameter here https://github.com/downshift-js/downshift#children-function */
+    onChange?: (selectedItem?: React.ElementType | null, stateAndHelpers?: any) => void;
+    items?: ItemPropType[];
+    theme?: {
+        colors?: object;
+        spacing?: object;
+        baseFontSize?: number;
+    };
```

Comment:
In Dropdown, we have a new prop called `ignoreSpecialChars `.
Merged this week.
How about of put this prop here?

Reply:
Done!